ntbs-service/Services/MockSpecimenService.cs

```diff
@@ -12,6 +12,11 @@ public class MockSpecimenService : ISpecimenService
         private readonly string _tbServiceWithResults;
         private readonly string _phecWithResults;
 
+        public static readonly int MockSpecimenNotificationId1 = 10100;
```

Comment:
Issue there is in the source of the assembly, MockSpecimenService is created in ntbs-service (for use when access to a hosted database is either impossible, or not necessary).

`ntbs-integration-tests` (site of `Utilities.cs`) assembly has a reference for `ntbs-service`, but not the other way round... so this was a result of necessity. 
It may be preferable to define a mockSpecimenService in integration testing, which would allow the ids to be placed consistently, but then we would have two versions of the mock which isn't too bad, but I kept it consistent for this changeset. Lemme know if you would prefer I create a mock service in integration testing.